src/operator/tensor/dot-inl.h

```diff
@@ -527,6 +533,78 @@ struct DotCsrTransRspRspByRowBlocks {
   }
 };
 
+/*!
+ * \brief CPU Kernel of PopulateCsrForNNC
+ * Parallelization by individual rows
+ */
+struct PopulateCsrForNNC {
```

Comment:
Can you add brief description on what this kernel is for?

Reply:
Added.